TYPO3.Flow/Classes/TYPO3/Flow/Package.php

```diff
@@ -86,14 +86,14 @@ public function boot(Core\Bootstrap $bootstrap)
                 return;
             }
             $objectManager = $bootstrap->getObjectManager();
-            $resourceManager = $objectManager->get(\TYPO3\Flow\Resource\ResourceManager::class);
+            $resourceManager = $objectManager->get(\TYPO3\Flow\ResourceManagement\ResourceManager::class);
```

Comment:
The namespaces should be relative (mhm .. thought I have done that already ....)